homeassistant/components/panel_iframe/__init__.py

```diff
@@ -37,18 +40,59 @@
     extra=vol.ALLOW_EXTRA,
 )
 
+STORAGE_KEY = DOMAIN
+STORAGE_VERSION_MAJOR = 1
+
 
 async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     """Set up the iFrame frontend panels."""
+    async_create_issue(
+        hass,
+        HOMEASSISTANT_DOMAIN,
+        f"deprecated_yaml_{DOMAIN}",
+        breaks_in_ha_version="2024.10.0",
+        is_fixable=False,
+        is_persistent=False,
+        issue_domain=DOMAIN,
+        severity=IssueSeverity.WARNING,
+        translation_key="deprecated_yaml",
+        translation_placeholders={
+            "domain": DOMAIN,
+            "integration_title": "map",
```

Comment:
Is `map` intentional here?

Reply:
Nope, that's a copy-paste mistake. Well spotted 👍